doc/userguide/rules/smb-keywords.rst

```diff
@@ -0,0 +1,12 @@
+SMB Keywords
+==============
+
+smb.fingerprint
+---------------
+
+Match on a fingerprint of a SMB client, kind of like ssh.hassh or TLS ja3.
```

Comment:
Do we want to include the size restriction for the SMB fingerprint?

Do you want to mention that `smb.fingerprint-client` is an alias?

Reply:
> Do we want to include the size restriction for the SMB fingerprint?

Which size restriction do you mean ?

> Do you want to mention that smb.fingerprint-client is an alias?

If you want me to ;-)